website/tables/aws/aws_dynamodbstreams_streams.md

```diff
@@ -0,0 +1,28 @@
+# Table: aws_dynamodbstreams_streams
+
+This table shows data for Dynamodbstreams Streams.
```

Comment:
This can be fixed by adding a `dynamodbstreams` entry to the `awsExceptions` map in `resources/plugin.go`